oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/common/JDBCTableInstaller.java

```diff
@@ -86,15 +88,16 @@ public boolean isExists(Model model) {
     @Override
     @SneakyThrows
     public void createTable(Model model) {
-        final var table = TableHelper.getLatestTableForWrite(model);
-        createTable(model, table);
+        final var dayTimeBucket = TimeBucket.getTimeBucket(System.currentTimeMillis(), DownSampling.Day);
+        createTable(model, dayTimeBucket);
     }
 
     @SneakyThrows
-    public void createTable(Model model, String table) {
+    public void createTable(Model model, long timeBucket) {
+        final var table = TableHelper.getTable(model, timeBucket);
         createOrUpdateTable(table, model.getColumns(), false);
         createOrUpdateTableIndexes(table, model.getColumns(), false);
-        createAdditionalTable(model);
+        createAdditionalTable(model, timeBucket);
```

Comment:
`createAdditionalTable` always used the current day timebucket to create a new table, when the `JDBCHistoryDeleteDAO` is creating tables for next day, the additional tables won't be created